x-pack/plugins/fields_metadata/common/fields_metadata/models/field_metadata.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import pick from 'lodash/pick';
+import { FieldAttribute, FieldMetadataPlain, PartialFieldMetadataPlain } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
nit: Can you please also add a comment above as to why the ES Lint exception is in place. This helps in future to navigate why a certain line was added